src/main/java/com/iexec/core/stdout/ReplicateStdout.java

```diff
@@ -0,0 +1,16 @@
+package com.iexec.core.stdout;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ReplicateStdout {
+    
```

Comment:
I would probably add chainTaskId so we keep the context inside the object

Reply:
I thought about that but I estimated it redundant.